### PR TITLE
Changes to update LAMMPS from PC2.x to PC3.x - close #35

### LINKED ISSUE #35: Add AWS ParallelCluster 3 config file for LAMMPS

### PR DESCRIPTION
**Pull request by jyothibv**
Signed-off-by Jyothi Venkatesh <jyothibv@amazon.com>

*Issue #, if available:* #35

*Description of changes:* Updates to LAMMPS to upgrade from PC2.x to PC3.x

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/lammps/scripts/setup/create_parallelcluster_config.sh [hunk 1]
```diff
@@ -129,13 +129,15 @@ else
     return 1
 fi
 
+# Install yq
```
Comment (mhuguesaws) [inline apps/lammps/scripts/setup/create_parallelcluster_config.sh:132]: `yq` installation should be moved into the pre-req installation script.

File: apps/lammps/scripts/setup/create_parallelcluster_config.sh [hunk 2]
```diff
@@ -148,24 +150,21 @@ else
...
-# Change the cluster configuration file
-PARALLELCLUSTER_CONFIG="${PARENT_PATH}/../../config/lammps-x86-64.ini"
-
-crudini --set ${PARALLELCLUSTER_CONFIG} "aws" aws_region_name "${AWS_REGION}"
-crudini --set ${PARALLELCLUSTER_CONFIG} "vpc public" vpc_id "${VPC_ID}"
-crudini --set ${PARALLELCLUSTER_CONFIG} "vpc public" master_subnet_id "${SUBNET_ID}"
-crudini --set ${PARALLELCLUSTER_CONFIG} "cluster default" key_name "${SSH_KEY_NAME}"
-crudini --set ${PARALLELCLUSTER_CONFIG} "cluster default" custom_ami "${LAMMPS_AMI}"
-crudini --set ${PARALLELCLUSTER_CONFIG} "fsx parallel-fs" import_path "s3://${BUCKET_NAME_DATA}"
+yq -i '.SharedStorage[1].FsxLustreSettings.ImportPath = strenv(BUCKET_URL)' ${PARALLELCLUSTER_CONFIG}
```
Comment (mhuguesaws) [inline apps/lammps/scripts/setup/create_parallelcluster_config.sh:165]: The cleanup script should be updated as well in order to read the new configuration file in yaml using `yq`.

**Review by mhuguesaws — COMMENTED** [review]
Thank you very much for your contribution. It helps bring LAMMPS on AWS ParallelCluster 3.x.

Please see my comments.

File: apps/lammps/scripts/setup/create_parallelcluster_config.sh (hunk 2 above)
Comment (jyothibv) [reply apps/lammps/scripts/setup/create_parallelcluster_config.sh:165]: Good catch. Thank you. Done!

**Review by mhuguesaws — APPROVED** [review]
Thank you for addressing the comments quickly.

LGTM.

**Pull request merged**